components/views/save.js

```diff
@@ -1,108 +1,18 @@
-import React, { useContext, useState, useEffect } from "react";
-import { Text, View, Image, FlatList, TouchableOpacity } from "react-native";
-import { Chip } from "react-native-elements/dist/buttons/Chip";
-import GradientText from "../colors/gradient-text";
-import Database from "../database";
-import saveStyle from "../styles/save-style";
-import AppContext from "../AppContext";
+import React from "react";
 import PropTypes from "prop-types";
+import CardList from "../cardList";
 
-const RenderSaveScreen = ({ item, username }) => (
-  <View style={saveStyle.card}>
-    <Image
-      source={{ uri: item.images[0] }}
-      resizeMode="cover"
-      overflow="hidden"
-      style={saveStyle.image}
-    />
-    <View style={saveStyle.removeBox}>
-      <TouchableOpacity
-        style={saveStyle.remove}
-        onPress={() => {
-          Database.removeSaved(item.id, username);
-        }}
-      />
-    </View>
-    <View style={saveStyle.cardContent}>
-      <Text style={saveStyle.name}>{item.name}</Text>
+export default function SearchScreen({ navigation }) {
```

Comment:
Cool but in saved all the cards just need to have "star" as icon. It doesn't need the start outline part.

Reply:
But then again. since you filter then i guess they just have a filled star

Reply:
Not sure how to change that :b